brainkeeper/src/app/components/session_screen/person-name/person-name.component.scss

```
@@ -0,0 +1,5 @@
+.container {
+    font-family: work snas;
```

it would neet to be

```suggestion
    font-family: Work Sans;
```

but i think that should be default anyway, after we merge the font branch from @sschoetz